backend/tests/v5/unit/processors/teamspaces/projects/projects.test.js

```diff
@@ -77,6 +84,47 @@ const testGetProjectList = () => {
 	});
 };
 
+const testDeleteProject = () => {
+	describe('Delete a project', () => {
+		test('should delete a project with no models', async () => {
+			await Projects.deleteProject('teamspace', '3');
+			expect(getProjectByIdMock.mock.calls.length).toEqual(1);
```

Comment:
Not really request for change (unless you want to)

But I recently found out that you could use [`toHaveBeenCalledWith()`](https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-) to test the arguments if you know exactly what they should be 😆 

Just thought you might want to know lol